packages/ndla-accordion/src/Accordions.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState, Children } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+// @ts-ignore
+import Button from '@ndla/button';
+
+import { getPanelIds, getOpenPanels } from './accordionUtil';
+import AccordionSection from './AccordionSection';
+
+type Props = { tiny?: boolean; single?: boolean };
+
+const Accordions: React.FC<Props & tType> = ({ tiny, single, t, children }) => {
```

Comment:
Synes endringa som @jnatten hadde i ed var fiffig. Var nokke sånt, men må sjekkes om kompilerer.
```suggestion
const Accordions = ({ tiny, single, t, children }: Props & tType) => {
```
Då unngår vi ha å med FC, som @N4G1 meinte var fraråda.

Reply:
Forskjellen er at da må man definere children selv i Props